css/skeleton.css

```diff
@@ -416,3 +416,146 @@ there.
 
 /* Larger than Desktop HD */
 @media (min-width: 1200px) {}
+
+/* Custom Bootstrap Buttons
+–––––––––––––––––––––––––––––––––––––––––––––––––– */
+
+/*
+This are just a set of button styles that follow nearly the same concept of that
+of Bootstrap, added by LokotamaTheMastermind
+*/
+
+/* Button type -> Warning
+––––––––––––––––––––––––––––––––––––––––––––––––––
+Bootstrap based: btn btn-warning-outline */
+
```

Comment:
do we really need these buttons ?

Reply:
I just realised that, I'm gonna close this PR anyway